resolution/controllers/hub_key_handler.py

```diff
@@ -167,7 +164,10 @@ def resolve_link_id_type(reference_links, parsed_key):
     link_for_id_type = None
```

Comment:
1. resolution did not have a model - and indeed does not really have "models" since it has no db to back it - it is hence fairly reasonable that the logic ends up in controller. Also this pattern of having the utility functions for the controllers the controller file for resolution, has been initiated in previous commits by different members of the team. We can move this to another file - it is not really clear that it will add value at this stage.

2. If the comment is formal and if you think the current implementation is difficult to read, it is possible to introduce another utility function. If the comments relates to the actual engineering logic, let me know if you think we shall simplify the process. So far, this I believe it is what has been understood from the requirements and from latest "reference_links" implementation.